cypress/plugins/testDataProcessor.js

```diff
@@ -50,16 +55,47 @@ function testDataProcessor(configEnv) {
   ]);
 
   // Resolving the variables in the JSON
-  const resolvedFireboltCallsJson = processFireboltJson(combinedFireboltCallsJson);
+  resolvedFireboltCallsJson = processFireboltJson(combinedFireboltCallsJson);
+
+  // Resolving the variables in the SetResponse JSON
+  const resolvedSetResponseJson = processSetResponseJson(mergedSetResponseJson);
```

Comment:
Inside the set response glue code we are parsing the key name, make sure to remove the parsing from glue code and also check can we able to use the parsed data in beforeOperation(moduleReqId Json)

Reply:
Made changes in beforeOperation to use the parsed data